src/main/java/io/confluent/connect/elasticsearch/ConfigCallbackHandler.java

```diff
@@ -194,8 +177,13 @@ private PoolingNHttpClientConnectionManager configureConnectionManager() {
         cm = new PoolingNHttpClientConnectionManager(ioReactor);
       }
 
-      cm.setDefaultMaxPerRoute(config.maxInFlightRequests());
-      cm.setMaxTotal(config.maxInFlightRequests());
+      int maxPerRoute = Math.max(10, config.maxInFlightRequests() * 2);
```

Comment:
@erdody could you comment on the new logic for the maxPerRoute and maxTotal calculation?